paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationInterceptorKtx.kt

```diff
@@ -28,7 +33,19 @@ internal suspend fun IntentConfirmationInterceptor.intercept(
             )
         }
         else -> {
-            error("Attempting to confirm intent for invalid payment selection: $paymentSelection")
+            val exception =
```

Comment:
Should we move this code into `DefaultIntentConfirmationInterceptor`, where we have a reference to the `Context`?

Reply:
I'm not sure I'm following. We don't have a way to call `DefaultIntentConfirmationInterceptor` in this case, since we don't know the params we'd send.